tests/Makefile.am

```diff
@@ -1,18 +1,13 @@
 AUTOMAKE_OPTIONS = foreign
 check_PROGRAMS = example_kem speed_kem test_kem \
                       minimal_kex_oqs test_kex \
-                      test_sig \
+                      minimal_sig_oqs test_sig \
```

Comment:
I'd prefer to use the name `example_sig` instead of `minimal_sig_oqs`, to mirror what we've done for KEMs and as on nist-branch.

Reply:
Ah yes, I was wondering if there would be a minimal_kem_oqs to replace the kex one. I can rename that one.

Reply:
I migrated the example_sig from the nist-branch.